source/tyr/tyr/helper.py

```diff
@@ -251,7 +246,9 @@ def load_instance_config(instance_name):
     instance.pg_username = config['database']['username']
     instance.pg_password = config['database']['password']
     instance.pg_port = int(config['database']['port'])
-    create_repositories(instance)
+
+    create_repositories([instance.source_directory, instance.backup_directory], instance.name)
+    create_repositories([os.path.dirname(instance.target_file)], instance.name)
```

Comment:
all minors:
1. The `os.path.dirname()` is useless with the use of `os.makedirs`, no ?

2. You could group :
```suggestion
    create_repositories([instance.source_directory, instance.backup_directory, instance.target_file], instance.name)
```

3. My understanding is that it's a "silent" break if source_dir, target_dir or backup_dir are missing (None) in instance: is that OK ?

Reply:
1- No, taget_file is file /path/data.nav.lz4
2- I can't because of a taget_file
3- Yes